nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1486,35 +1588,35 @@ If no administrator action is taken, the configuration values remain unencrypted
 [[encrypt-config_tool]]
 === Encrypt-Config Tool
 
-The `encrypt-config` command line tool (invoked as `./bin/encrypt-config.sh` or `bin\encrypt-config.bat`) reads from a 'nifi.properties' file with plaintext sensitive configuration values, prompts for a master password or raw hexadecimal key, and encrypts each value. It replaces the plain values with the protected value in the same file, or writes to a new 'nifi.properties' file if specified.
+The `encrypt-config` command line tool (invoked as `./bin/encrypt-config.sh` or `bin\encrypt-config.bat`) reads from a _nifi.properties_ file with plaintext sensitive configuration values, prompts for a master password or raw hexadecimal key, and encrypts each value. It replaces the plain values with the protected value in the same file, or writes to a new _nifi.properties_ file if specified.
```

Comment:
L1454 The link for `Bcrypt Spring Security` is broken. It has to be updated to https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/crypto/bcrypt/BCrypt.html

Reply:
Good catch.  Will update.